fontbe/src/gvar.rs

```diff
@@ -0,0 +1,101 @@
+//! Generates a [gvar](https://learn.microsoft.com/en-us/typography/opentype/spec/gvar) table.
+
+use std::cmp::{min, max};
+
+use font_types::{F2Dot14, GlyphId};
+use fontdrasil::orchestration::Work;
+use fontir::coords::NormalizedLocation;
+use write_fonts::{
+    dump_table,
+    tables::{
+        gvar::{GlyphDeltas, GlyphVariations, Gvar},
+        variations::Tuple,
+    },
+};
+
+use crate::{
+    error::Error,
+    orchestration::{BeWork, Bytes, Context, GvarFragment},
+};
+
+struct GvarWork {}
+
+pub fn create_gvar_work() -> Box<BeWork> {
+    Box::new(GvarWork {})
+}
+
+fn to_tuple(loc: &NormalizedLocation) -> Tuple {
```

Comment:
can this be a method on `NormalizedLocation`?

Reply:
no, because I don't think a NormalizedLocation by itself has enough information to fully define a Tuple's start/peak/end region, it's only the peak. Unless the intention is to construct such an incomplete Tuple and modify afterwards

Reply:
It's completely not the right way to compute the Tuple, let us pretend it never existed :D